src/applications/personalization/profile/tests/e2e/address-validation/two-suggestions.cypress.spec.js

```diff
@@ -0,0 +1,50 @@
+import { setUp } from 'applications/personalization/profile/tests/e2e/address-validation/setup';
+
+describe('Personal and contact information', () => {
+  describe('when entering a home address that has two suggestions', () => {
+    it('should successfully update on Desktop', () => {
+      setUp('two-suggestions');
+
+      cy.get('#root_addressLine1')
```

Comment:
Since these tests are really all about making sure the front end handles different responses from the address validation API, we _could_ input the same address into the form for every single test.